src/main/java/redis/clients/util/JedisClusterHashTagUtil.java

```diff
@@ -14,6 +14,10 @@ public static String getHashTag(String key) {
     return extractHashTag(key, true);
   }
 
+  public static boolean isClusterCompliantMatchPattern(byte[] matchPattern) {
```

Comment:
Looks like this is unrelated to the actual changeset. Minimizing changeset helps us avoiding merge conflict.